float64 x_vel
float64 distance
uint8 cmd

---

bool success